### ISSUE TITLE
Feature request: `genie.query` should support a customized number of results

### ISSUE THREAD

**Issue by george1459** [stanford-oval/pyGenieScript#12]

**Comment by george1459** [comment]
Resolved by #13

**Comment by george1459** [comment]
```
>>> genie.query("show me a chinese restaurant")['response']
['I see Jing Jing Chinese Gourmet. It is a restaurant with the price range equal to moderate and phone number (650) 328-6885.']
>>> genie.query("show me a chinese restaurant", num_results=2)['response']
['I see Jing Jing Chinese Gourmet and Indochine. Both are Chinese restaurants rated 3.5 stars.']
>>> genie.query("show me a chinese restaurant", num_results=3)['response']
["I see Jing Jing Chinese Gourmet, Indochine, and Chef Chu's. They're Chinese moderate restaurants."]
```
